src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -120,6 +130,26 @@ def test_conversion_as_lone_interface(self):
                 self.run_custom_checks()
 
 
+def _create_mock_probe_for_recording(recording):
+    import probeinterface as pi
+    import spikeinterface as si
+
+    R: si.BaseRecording = recording
```

Comment:
What's on here? Why not just use a normal typehint?

```suggestion
def _create_mock_probe_for_recording(recording: si.BaseRecording):
    import probeinterface as pi
    import spikeinterface as si

    recording
```
Is it because you want passive loading of probeinterface and spikeinterface?

Reply:
@bendichter yes... I didn't want to create a dependency on spikeinterface at the module level.

Reply:
It looks like the only attribute you are using from the recording in the number of electrodes. Would it be better to just accept that as an input? Or do you want to allow yourself the flexibility to use more attributes from the RecordingExtractor object in the future?

Reply:
you're right. I have adjusted this so it only takes in the num_channels parameter. This simplifies the typing as well.